src/develop/ui/patterns/mobile/gallery.md

```diff
@@ -3,60 +3,49 @@ tags: runtime-mobileandreactiveweb;
 summary: 
 ---
 
-# Gallery Pattern
+# Gallery 
 
-The Gallery pattern
-displays content (such as cards) in a specific set of columns, configurable
-per device type and orientation. Use this pattern to display a List of
-elements side by side, with a different number of items per row on different
-devices.
+You can use the Gallery UI Pattern to display content (such as cards) in a specific set of columns, configurable per device type and orientation. 
 
-Here's a preview in Service Studio:
+![](<images/gallery-image-7.png>)
 
-![](images/Gallery_preview.png)
 
-## How to Use the Gallery Pattern
+## How to use the Gallery UI Pattern
 
-1\. Drag a list or static content.
+1. In Service Studio, in the Toolbox, search for `Gallery`. 
 
-2\. Set the number of items in Input Parameters.
+    The Gallery widget is displayed.
 
-### Using Animations Inside the Gallery
+    ![](<images/gallery-image-8.png>)
 
-1\. Use the **Animate** block as the first element inside the list.
+1. From the Toolbox, drag the Gallery widget onto your application’s screen.
 
-![](images/Gallery_animate.png)
 
-2\. Place your content inside the b block.
+1.  Drag a list or static content into the Content placeholder.
 
-![](images/Gallery_ellipsis.png)
+1. On the **Properties** tab, set the number of items.
 
-3\. Define the **Animate** block with your desired type of animation, and set
-the delay with current row number from the list.
+After following these steps and publishing the module, you can test the pattern in your app.
 
-![](images/Gallery_interaction.png)
+## Properties
 
-**Note**: If you are using a List inside a Gallery, you need to disable
```

Comment:
@annettewilliamsoutsystems @pacferreira this seems like an awfully important mention for it to disappear. Unless it's incorrect, it should be in a section at the end such as additional notes or Compatibility with Other Widgets or something.